### PR TITLE
Added Credit Section

### PR DESCRIPTION
**Pull request by rhampt**
Updated Billing docs to explain credits

Change-type: patch
Signed-off-by: Ryan Hampton (@rhampt) <rhampt@balena.io>

---

> *Please make sure to read the [CONTRIBUTING](https://github.com/balena-io/docs/blob/master/CONTRIBUTING.md) document before opening the PR for relevant information on contributing to the documentation. Thanks!*

### CODE REVIEW

File: pages/learn/manage/billing.md [hunk 1]
```diff
@@ -14,53 +15,75 @@ From this dashboard, you can change your billing plan, update your billing infor
...
 
 ## Inactive devices
 
 If you have devices that will be inactive for long periods of time, you can mark them as `Inactive`. This will keep them from being counted towards your device total. We’ll charge a one-time deactivation fee, and the devices will remain inactive until they come online.
 
-Marking a device as `Inactive` is a [device action][device-action]. Device actions can be applied in one of three ways: as a group action applied to one or more devices in the device list, as an action selected from the dropdown on the device dashboard, or directly from the *Actions* tab of the device dashboard. Once a device has been marked `Inactive`, it will be faded out in the device list.
+Marking a device as `Inactive` is a [device action][device-action]. Device actions can be applied in one of three ways: as a group action applied to one or more devices in the device list, as an action selected from the dropdown on the device dashboard, or directly from the _Actions_ tab of the device dashboard. Once a device has been marked `Inactive`, it will be faded out in the device list.
```
Comment (myarmolinsky) [inline pages/learn/manage/billing.md:86]: While you're here, it's not actually under the actions tab anymore and it's not considered an action. It is a `device setting` (not sure what to do with the `device-action` bit, maybe that stays the same?) under the `Settings` tab of the device dashboard

Comment (rhampt) [reply pages/learn/manage/billing.md:86]: @myarmolinsky Good find. I think I will include this update when the whole page is audited (I plan to do this soon)

**Comment by vipulgupta2048** [comment]
Will final review when images are added @rhampt

File: pages/learn/manage/billing.md (hunk 1 above)
Comment (rhampt) [reply pages/learn/manage/billing.md]: done

**Comment by rhampt** [comment]
@vipulgupta2048 Please provide a review when you are able. Thanks!

**Comment by rhampt** [comment]
I self-certify!

File: pages/learn/manage/billing.md [hunk 2]
```diff
@@ -44,27 +44,43 @@ Further down, you'll find a place to download invoices, add or edit payment meth
...
+
+This system is available to all non-Prototype customers. Our Pilot, Production, and Enterprise customers have full access to Credit discounts by paying in advance for device usage for any volume.
+
+To manage Credits for a specific organization, navigate to the organization and select the `Credits` tab from the sidebar on the left:
+
+<img src="/img/common/billing/credits_nav.png" width="25%">
+
+On the Credits page, you're able to see Credit purchase history and the number of Credits that have been purchased and are still allocatable for device usage. This is also where you can initiate Credit purchases. An invoice will be automatically generated and listed on the Billing page when a bulk Credit purchase is initiated.
+
+Credits will expire 10 years after the date of purchase. If there is no activity for 2 years we may at our discretion cancel any outstanding credits and will provide 60 days notice to the customer of our intention to do so. Credit purchases are non-refundable and cannot be refunded due to project or plan cancellation.
```
Comment (vipulgupta2048) [inline pages/learn/manage/billing.md:61]: Are credits transferable?

Comment (rhampt) [reply pages/learn/manage/billing.md:61]: They are not at the moment, maybe between orgs later. Good catch, I'll add this now @vipulgupta2048

**Review by vipulgupta2048 — APPROVED** [review]

File: pages/learn/manage/billing.md (hunk 2 above)
Comment (vipulgupta2048) [reply pages/learn/manage/billing.md:61]: Yeah, it's what my mind asked? So I thought I drop it in even if it is nitpick. Solid PR overall, approved it.

**Comment by vipulgupta2048** [comment]
Yeah, i think my approval flipped the auto-merge back on in the system @rhampt

**Comment by rhampt** [comment]
Started a new PR for Credit doc updates. I'm sure there will be more when customers start giving feedback. Here is is for posterity: https://github.com/balena-io/docs/pull/2494

**Pull request merged**